nuxeo-core/nuxeo-core-storage-sql/nuxeo-core-storage-sql/src/main/java/org/nuxeo/ecm/core/storage/sql/jdbc/dialect/DialectSQLServer.java

```diff
@@ -440,14 +441,12 @@ public FulltextMatchInfo getFulltextScoredMatchInfo(String fulltextQuery, String
             info.joins.add(new Join(Join.LEFT, ft.getQuotedName(), null, null, ftMain.getFullQuotedName(),
                     mainColumn.getFullQuotedName()));
         }
-        info.joins.add(
-                new Join(Join.LEFT, //
-                        String.format("CONTAINSTABLE(%s, *, ?, LANGUAGE %s)", ft.getQuotedName(),
-                                getQuotedFulltextAnalyzer()),
-                        tableAlias, // alias
-                        fulltextQuery, // param
-                        ftMain.getFullQuotedName(), // on1
-                        String.format("%s.[KEY]", tableAlias) // on2
+        info.joins.add(new Join(Join.LEFT, //
```

Comment:
Could you rollback this change as it's not related to the NXP? You can add a `//` after `add(` to prevent further formatting issues.